db/projects/moeda-loyalty-points/project.yaml

```diff
@@ -0,0 +1,58 @@
+name: Moeda Loyalty Points
+alt_names:
+  - Moeda
+web:
+  - 'https://moedaseeds.com/'
+assets:
+  - id: mda
+    name: Moeda Loyalty Points
+    type: token
+    token_standard: ERC-20
+    ledger: ethereum
+    symbol: MDA
+    contract_address: '0x51db5ad35c671a87207d88fc11d593ac0c8415bd'
+    decimals: 18
+    max_supply: 19628888
+    start_date: '2017-07-17'
+    ico_properties:
+      start_date: '2017-08-28'
+      end_date: '2017-09-28'
+      total_cap: 20000000
+      ico_price: 1 USD
+      country: Brazil
+    webids:
+      coinmarketcap: moeda-loyalty-points
+      cryptocompare: mda
+      cryptocompare/id: '341279'
+team:
+  - name: Taynaah Reis
+    role: ceo
+    photo: https://moedaseeds.com/assets/images/site/profiles/taynaah.jpg
+  - name: Brad Chun
+    role: co-founder
+    photo: https://d33wubrfki0l68.cloudfront.net/b2b86c1270b47a3b189840401f3b86ad6c040f72/26c43/assets/images/site/profiles/brad.jpg
+  - name: Isa Yu
+    role: co-founder
+    photo: https://d33wubrfki0l68.cloudfront.net/4f61180a491285ee8061ab436faf6b1380f31c57/0dd65/assets/images/site/profiles/isa.jpg
+  - name: Edward Stehling
+    role: director of operations
+    photo: https://moedaseeds.com/assets/images/site/profiles/edward-stehling.jpg
+  - name: Ubirajara Melo
+    role: cto
+    photo: https://moedaseeds.com/assets/images/site/profiles/ubirajara-melo.jpg
```

Comment:
We  should probably focus on any social links of teams, not just photo - we can always get images from their profiles

Reply:
webids:
      linkedin: 'taynaah-reis'
      medium: '@taynaahreis'

    webids:
      linkedin: 'isabella-yu-297212a3'
      medium: '@Isa.moeda'

    webids:
      github: 'ubirajara-infolog'